contracts/lazy_mint/ERC721/ILazyMintERC721.sol

```diff
@@ -95,6 +95,9 @@ interface ILazyMintERC721 {
     /// @dev Emitted when transfers are set as restricted / not-restricted.
     event TransfersRestricted(bool restricted);
 
+    /// @dev Emitted when a new Owner is set.
+    event NewOwner(address prevOwner, address newOwner);
```

Comment:
i wonder if we should keep this in LazyMintERC721 rather than the interface. nbd tho.